pithos/pithos.py

```diff
@@ -163,16 +163,17 @@ def __init__(self, app, test_mode):
 
         email = self.settings['email']
         try:
-            password = get_account_password(email)
+            test = get_account_password(email)
```

Comment:
__[flake8]__
* [F841] local variable 'test' is assigned to but never used

<!-- Comment by [SideCI](https://sideci.com) -->